emails/tests/fixtures/disabled_mask_for_spam_expected.email

```diff
@@ -0,0 +1,289 @@
+                margin-right: 0 !important;
+            }
+        }
+       =20
+        @media screen and (max-width: 768px) {
+            .footer-block {
+                display: block;
+                width: 100% !important;
+                text-align: left !important;
+            }
+            .footer-block a {
+                display: block;
+            }
+            img {
+                margin: 0 !important;
+            }
+        }
+
+        @media screen and (max-width: 480px) {
+            .header-block {
+                display: block;
+                width: 100% !important;
+                text-align: left !important;
+            }
+
+            .container-link {
+                margin-right: 0 !important;
+            }
+        }
+    </style>
+
+    <!-- temporary header -->
+    <table id=3D"relay-email-header" width=3D"100%" bgcolor=3D"#3D3D3D" style=
+=3D"background: #3D3D3D; padding: 12px; margin-bottom: 30px; margin-top: 30px=
+; width: 96%; border-radius: 6px; max-width: 1200px;" align=3D"center">
+        <tr>
+            <td width=3D"50%" class=3D"header-block" style=3D"line-height: 15=
+0%;" align=3D"left">=20
+                <img width=3D"155" src=3D"http://127.0.0.1:8000/static/images=
+/email-images/relay-logo-emails-dark-bg.png" style=3D"margin: 0 5px;" alt=3D"=
+warning icon"/>  =20
+            </td>
+            <td width=3D"50%" class=3D"header-block" align=3D"right">=20
+                <a class=3D"container-link" href=3D"http://127.0.0.1:8000/pre=
+mium/?utm_campaign=3Demail_replies&amp;utm_source=3Demail&amp;utm_medium=3Dem=
+ail" style=3D"color: white;">Upgrade for more protection</a>=20
+            </td>
+        </tr>
+    </table> =20
+
+
+    <table role=3D"presentation" border=3D"0" cellpadding=3D"0" cellspacing=
+=3D"10px" style=3D"padding: 60px 30px 120px 30px;" align=3D"center">
+        <tr>
+            <td style=3D"max-width:850px; padding-top: 0px; padding-bottom: 0=
+px; text-align: center;">
+               =20
+                <h2 style=3D"font-family: inter medium, Arial, system-ui, san=
+s-serif;">=20
+                    <img width=3D"18" src=3D"http://127.0.0.1:8000/static/ima=
+ges/email-images/warning.png" style=3D"margin: 0 5px;" alt=3D"warning icon"/>
+                    =E2=81=A8Firefox Relay=E2=81=A9 has disabled one of your =
+email masks.
+                </h2>
+                <p style=3D"line-height: 200%; margin-bottom: 30px; padding: =
+20px 60px 20px 60px">
+                Firefox Relay received a spam complaint for an email sent to =
+<strong>w41fwbt4q@test.com</strong>. This usually happens if you or your emai=
+l provider mark an email as spam. To prevent further spam, Firefox Relay has =
+disabled your <strong>w41fwbt4q@test.com</strong> mask.
+                </p>=20
+                <a href=3D"http://127.0.0.1:8000/accounts/profile/#w41fwbt4q%=
+40test.com">
+                    Visit your =E2=81=A8Firefox Relay=E2=81=A9 dashboard to r=
```

Comment:
Here's what the FSI and PDI codepoints look like in the HTML. The `convert_fsi_to_span` filter changes these to `<span dir="auto">...</span>`, but it can be a pain to use.